.circleci/regenerate.py

```diff
@@ -260,6 +260,7 @@ def unittest_workflows(indentation=6):
                             }
                         }
                     )
+    jobs += docstring_parameters_sync_job(None)
```

Comment:
This will attach the job to all the unittest jobs, which is redundant. One job should do.